fury/ui.py

```diff
@@ -2161,10 +2164,18 @@ def __init__(self, line_width=5, inner_radius=0, outer_radius=10,
         self.shape = shape
         self.default_color = (1, 1, 1)
         self.active_color = (0, 0, 1)
+        self.orientation = orientation.lower()
         super(LineDoubleSlider2D, self).__init__()
 
-        self.track.width = length
-        self.track.height = line_width
+        if self.orientation == "horizontal":
+            self.track.width = length
+            self.track.height = line_width
+        elif self.orientation == "vertical":
+            self.track.width = line_width
+            self.track.height = length
+        else:
+            raise ValueError("Unknown orientation")
```

Comment:
Can you add a test for this using `npt.assert_raises` in `test_ui.py`

Reply:
![image](https://user-images.githubusercontent.com/29832615/76395903-3b5aa400-639e-11ea-9bb0-0bfb73d6b9d9.png)
Isn't this the correct way of doing it?

Reply:
Don't worry I figured it out :+1: